go.mod

```diff
@@ -228,6 +228,7 @@ require (
 )
 
 replace (
+	github.com/go-jose/go-jose/v4 => github.com/go-jose/go-jose/v4 v4.0.5 // CVE-2025-27144
```

Comment:
/hold

We should not do this replace
We just need let the tools work out and sync ypstrem with downstream,
Upstream is fixed already